src/test/java/com/danit/controllers/ClientControllerTest.java

```diff
@@ -160,4 +165,20 @@ public void deleteClientById() throws Exception {
     assertEquals(numberOfClients, clientService.getNumberOfClients());
 
   }
+
+  @Test
+  public void whenGetOnePageWithThreeItemsRequestedReturnsOnePageWithThreeItems() throws Exception {
+    HttpHeaders header = testUtils.getHeader(template, UserRolesEnum.USER);
+    mockMvc.perform(get("/clients?page=1&size=3").headers(header))
+        .andExpect(content().contentType(MediaType.APPLICATION_JSON_UTF8))
+        .andExpect(jsonPath("$.data[*]", hasSize(3)));
+  }
+
+  @Test
+  public void searchByNameReturnsValidResult() throws Exception {
+    HttpHeaders header = testUtils.getHeader(template, UserRolesEnum.USER);
```

Comment:
first you have to add entity and then get and check
imagine, in future there is no Camilla user in database what will happen with this test)?